config/settings.yml

```diff
@@ -357,6 +358,7 @@ defaults: &defaults
     lock_deployment_max_duration:  86400 # lock ttl for deployments in seconds. Its kept at 24 hrs - 60 * 60 * 24
     infrastructure: &director_infrastructure
       stemcell:
+        alias: service_fabrik_stemcell
```

Comment:
stemcells are common across landscape. Dont prefix the name as service-fabrik_stemcell. Rather can be something like 'ubunutu-trusty' [just check whatever is used in cf core.]

Reply:
ok.. I will be renaming the alias to 'ubuntu-trusty'